diff --git a/packages/server/src/adapters/node-http/incomingMessageToRequest.ts b/packages/server/src/adapters/node-http/incomingMessageToRequest.ts
index 3a590524c..4223db4ca 100644
--- a/packages/server/src/adapters/node-http/incomingMessageToRequest.ts
+++ b/packages/server/src/adapters/node-http/incomingMessageToRequest.ts
@@ -1,4 +1,5 @@
 import type * as http from 'http';
+import { IncomingMessage } from 'node:http';
 import { TRPCError } from '../../@trpc/server';
 import type { NodeHTTPRequest, NodeHTTPResponse } from './types';
 
@@ -21,6 +22,10 @@ function createBody(
     if (typeof req.body === 'string') {
       return req.body;
     }
+    // formData use
+    if (req.body instanceof IncomingMessage) {
+      return req.body as any;
+    }
     // If body exists but isn't a string, stringify it as JSON
     return JSON.stringify(req.body);
   }
