diff --git a/packages/server/src/unstable-core-do-not-import/stream/utils/asyncIterable.ts b/packages/server/src/unstable-core-do-not-import/stream/utils/asyncIterable.ts
index afe6a96c3..aaa40e56f 100644
--- a/packages/server/src/unstable-core-do-not-import/stream/utils/asyncIterable.ts
+++ b/packages/server/src/unstable-core-do-not-import/stream/utils/asyncIterable.ts
@@ -8,6 +8,12 @@ export function iteratorResource<TYield, TReturn, TNext>(
 ): AsyncIterator<TYield, TReturn, TNext> & AsyncDisposable {
   const iterator = iterable[Symbol.asyncIterator]();
 
+  // @ts-expect-error - this is added in node 24 which we don't officially support yet
+  // eslint-disable-next-line no-restricted-syntax
+  if (iterator[Symbol.asyncDispose]) {
+    return iterator as AsyncIterator<TYield, TReturn, TNext> & AsyncDisposable;
+  }
+
   return makeAsyncResource(iterator, async () => {
     await iterator.return?.();
   });
