diff --git a/Sources/NIOFS/Internal/SystemFileHandle.swift b/Sources/NIOFS/Internal/SystemFileHandle.swift
index d99de21e..0ccd0400 100644
--- a/Sources/NIOFS/Internal/SystemFileHandle.swift
+++ b/Sources/NIOFS/Internal/SystemFileHandle.swift
@@ -848,6 +848,8 @@ extension SystemFileHandle.SendableView {
                         flags: materialization.exclusive ? [.exclusive] : []
                     )
                 }
+                #elseif os(Windows)
+                fatalError("file materialization via rename is unavailable on Windows")
                 #endif
 
                 if materialization.exclusive {
