diff --git a/Sources/NIO/Selector.swift b/Sources/NIO/Selector.swift
index 843ac58e..3dc5da2f 100644
--- a/Sources/NIO/Selector.swift
+++ b/Sources/NIO/Selector.swift
@@ -331,6 +331,8 @@ final class Selector<R: Registration> {
          is likely to cause performance problems. By abusing ARC, we get the guarantee that there won't be any future
          wakeup calls as there are no references to this selector left. 💁
          */
+
+        // we try! this because `close` only fails in cases that should never happen (EBADF).
 #if os(Linux)
         try! Posix.close(descriptor: self.eventfd)
 #else
