diff --git a/Tests/ComposableArchitectureTests/CurrentValueRelayTests.swift b/Tests/ComposableArchitectureTests/CurrentValueRelayTests.swift
new file mode 100644
index 00000000..98099496
--- /dev/null
+++ b/Tests/ComposableArchitectureTests/CurrentValueRelayTests.swift
@@ -0,0 +1,29 @@
+#if DEBUG
+@preconcurrency import Combine
+@testable @preconcurrency import ComposableArchitecture
+import XCTest
+
+final class CurrentValueRelayTests: BaseTCATestCase {
+  func testConcurrentSend() async {
+    nonisolated(unsafe) let subject = CurrentValueRelay(0)
+    let values = LockIsolated<Set<Int>>([])
+    let cancellable = subject.sink { (value: Int) in
+      values.withValue {
+        _ = $0.insert(value)
+      }
+    }
+
+    await withTaskGroup(of: Void.self) { group in
+      for index in 1...1_000 {
+        group.addTask {
+          subject.send(index)
+        }
+      }
+    }
+
+    XCTAssertEqual(values.value, Set(Array(0...1_000)))
+
+    _ = cancellable
+  }
+}
+#endif
