diff --git a/httpx/_compat.py b/httpx/_compat.py
new file mode 100644
index 0000000..47c12ba
--- /dev/null
+++ b/httpx/_compat.py
@@ -0,0 +1,6 @@
+# `contextlib.asynccontextmanager` exists from Python 3.7 onwards.
+# For 3.6 we require the `async_generator` package for a backported version.
+try:
+    from contextlib import asynccontextmanager  # type: ignore
+except ImportError:  # pragma: no cover
+    from async_generator import asynccontextmanager  # type: ignore # noqa
