diff --git a/tests/test_cli.py b/tests/test_cli.py
index 79de1fc8..09995488 100644
--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -679,3 +679,8 @@ def test_cli_empty(app):
 
     result = app.test_cli_runner().invoke(args=["blue", "--help"])
     assert result.exit_code == 2, f"Unexpected success:\n\n{result.output}"
+
+
+def test_run_exclude_patterns():
+    ctx = run_command.make_context("run", ["--exclude-patterns", __file__])
+    assert ctx.params["exclude_patterns"] == [__file__]
