diff --git a/ignore/src/overrides.rs b/ignore/src/overrides.rs
index c737b04..faf05f3 100644
--- a/ignore/src/overrides.rs
+++ b/ignore/src/overrides.rs
@@ -214,4 +214,10 @@ mod tests {
         assert!(ov.matched("src/foo", false).is_ignore());
         assert!(ov.matched("src/foo", true).is_none());
     }
+
+    #[test]
+    fn absolute_path() {
+        let ov = ov(&["!/bar"]);
+        assert!(ov.matched("./foo/bar", false).is_none());
+    }
 }
