diff --git a/lib/faker/default/internet.rb b/lib/faker/default/internet.rb
index 9bbdb79a..09447f67 100644
--- a/lib/faker/default/internet.rb
+++ b/lib/faker/default/internet.rb
@@ -192,6 +192,14 @@ module Faker
         sample(agents)
       end
 
+      def uuid
+        # borrowed from: https://github.com/ruby/ruby/blob/d48783bb0236db505fe1205d1d9822309de53a36/lib/securerandom.rb#L250
+        ary = Faker::Config.random.bytes(16).unpack('NnnnnN')
+        ary[2] = (ary[2] & 0x0fff) | 0x4000
+        ary[3] = (ary[3] & 0x3fff) | 0x8000
+        '%08x-%04x-%04x-%04x-%04x%08x' % ary # rubocop:disable Style/FormatString
+      end
+
       alias user_name username
     end
   end
